Multisig/Logic/Models/PrivateKey/web3swift/BIP32HDNode.swift

```diff
@@ -235,7 +235,7 @@ extension HDNode {
             let newNode = HDNode()
             newNode.chaincode = cc
             newNode.depth = self.depth + 1
-            newNode.publicKey = pubKeyCandidate
+            newNode.publicKey = newPublicKey
```

Comment:
This is a bug which is already fixed in the `web3swift` library itself, please see detail here: https://github.com/skywinder/web3swift/pull/423